src/main/java/com/pollistics/models/Poll.java

```diff
@@ -16,22 +16,46 @@
 	private String name;
 	private HashMap<String,Integer> options;
 	private User user;
+	private String slug;
+
+	// todo: orden these constructors with `this(...arg)`
 
 	public Poll(String name, HashMap<String,Integer> options) {
 		this.name = name;
 		this.options = options;
+		this.slug = createSlug();
 	}
 
-	public Poll(String name, HashMap<String,Integer> options, User user) {
+	private String createSlug() {
+		// random three-word combo
+		return "meme-meme-meme";
```

Comment:
How should we create the slug; a method that fetches them out of a wordlist somewhere, then checks if it already exists. Maybe a different collection or something to keep it performant? I'm not so sure...

cc @eliasmeire @thysruben